darts/models/forecasting/forecasting_model.py

```diff
@@ -610,7 +622,7 @@ def historical_forecasts(
             `min_train_series_length`.
         start
```

Comment:
can we update the description for the integer start as well?
Maybe we change the start to something like this:

```
            Optionally, the first point in time at which a prediction is computed. This parameter supports:
            ``float``, ``int``, ``pandas.Timestamp``, and ``None``.
            If a ``float``, it is the proportion of the time series that should lie before the first prediction point.
            If an ``int``, it is either the index position of the first prediction point for `series` with a
            `pd.DatetimeIndex`, or the index value for `series` with a `pd.RangeIndex`. The latter can be changed to 
            the index position with `start_format="position"`.
            If a ``pandas.Timestamp``, it is the time stamp of the first prediction point.
            If ``None``, the first prediction point will automatically be set to:
```